docs/source/overview/jobs.rst

```diff
@@ -0,0 +1,137 @@
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _jobs:
+
+*************************
+Content Invalidation Jobs
+*************************
+:dfn:`Content Invalidation Jobs`, or simply "jobs" as they are sometimes known, are ways of forcing :term:`cache servers` to treat content as no longer valid, bypassing their normal caching policies.
+
+In general, this *should* be unnecessary, because a well-behaved :term:`Origin` *should* be setting its HTTP caching headers properly, so that content is only considered valid for some appropriate time intervals. Occasionally, however, an :term:`Origin` will be too optimistic with its caching instructions, and when content needs to be updated, :term:`cache servers` need to be informed that they must check back with the :term:`Origin`. Content Invalidation Jobs allow this to be done for specific patterns of assets, so that :term:`cache servers` will check back in with the :term:`Origin` and verify that the content they have cached is still valid.
+
+The model for Content Invalidation Job as API objects is given in :ref:`jobs-model` as a TypeScript interface.
+
+.. _jobs-model:
+
+.. code-block:: typescript
+	:caption: Content Invalidation Job as a Typescript interface.
+
+	/** This is the form used to create a new Content Invalidation Job */
+	interface ContentInvalidationJobCreationRequest {
+		deliveryService: string;
+		invalidationType: "REFRESH" | "REFETCH";
+		regex: `/${string}` | `\\/${string}`; // must also be a valid RegExp
+		startTime: Date; // RFC3339 string
+		ttlHours: number;
+	}
+
+	/**
+	 * This is the form used to return representations of Content Invalidation
+	 * Requests to clients.
+	 */
+	interface ContentInvalidationJob {
+		assetUrl: string;
+		createdBy: string;
+		deliveryService: string;
+		id: number;
+		invalidationType: "REFRESH" | "REFETCH";
+		startTime: Date; // RFC3339 string
+		ttlHours: number;
+	}
+
+.. _job-asset-url:
+
+Asset URL
+---------
+This property only appears in responses from the :ref:`to-api`, and has no effect in requests. The :dfn:`Asset URL` is constructed from the `Regular Expression`_ used in the creation of a Content Invalidation Job and the :ref:`ds-origin-url` of the :term:`Delivery Service` for which it was created. It is a URL that has a valid regular expression as its path (and may not be "percent-encoded" where a normal URL typically would be). Requests from CDN clients for content that matches this pattern will trigger Content Invalidation behavior.
+
+.. _job-created-by:
+
+Created By
+----------
+The username of the user who created the Content Invalidation Job is stored as the :dfn:`Created By` property of the Content Invalidation Job.
```

Comment:
Not sure if you want to add the `:term:` for both _Content Invalidation Job_ here, or throughout this page really. Given how consistent it is used throughout the API docs and elsewhere I thought I'd bring it up, however I am fine with it either way here.

Reply:
In general I don't use `term` for the overview section that describes the term. The flow of understanding for a term with an overview section is

1. See the term
1. Click on it, which takes you to a one-paragraph (max) glossary entry explanation
1. Click on the seealso link to go to the overview section if you need more detailed information

doesn't make sense to me to put a loop in that flow.